Packs/Listmonk/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "listmonk",
+    "description": "",
```

Comment:
Could you provide a description?
[for information on description, click here](https://xsoar.pan.dev/docs/documentation/pack-docs#pack-description:~:text=and%20README%3A-,Pack%20Description%23,-The%20pack%20description)